backend/routes/emergency.route.js

```diff
@@ -3,8 +3,12 @@ const router = express.Router();
 const authUtils = require('../utils/auth.js');
 const EmergencyController = require('../controllers/emergency.controller.js');
 
-router.get('/', authUtils.isAuthenticated, EmergencyController.getUserEmergency);
+
+router.get('/', authUtils.auth, EmergencyController.getUserEmergency);
```

Comment:
the function is called isAuthenticated, not auth, change it back to authUtils.isAuthenticated pls.
.